docs/tokens.md

```diff
@@ -66,13 +62,21 @@ TO DO
 ## Wallet
 
 TO DO
-## Issue and Transfer Tokens
+## Issuing and Transferring Tokens
 
 **TODO who are these examples meant for?**
+**TODO It looks like alice is giving carl normal tokens?**
+**TODO what is this Origin::signed business? Go to definiton takes me to the construct_runtime macro call** 
+**TODO should explain the index argument in tx.outpoint(1)**
+**TOOD: In the definition of MLT unit, it's 1000 * 100_000_000. Why written this way. One hundred billion**
```

Comment:
A rip-off of LN. Bitcoin breaks down to 100_000_000 satoshis. LN introduces the millisatoshi (sat/1000) hence the weird notation of 1_000 * 100_000_000

Reply:
Ah!